app/src/main/java/org/oppia/app/drawer/ExitProfileDialogFragment.kt

```diff
@@ -0,0 +1,70 @@
+package org.oppia.app.drawer
+
+import android.app.Dialog
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.view.ContextThemeWrapper
+import androidx.fragment.app.DialogFragment
+import org.oppia.app.R
+import org.oppia.app.profile.ProfileChooserActivity
+
+/** [DialogFragment] that gives option to either cancel or exit current profile */
```

Comment:
Nit full stop: `/** [DialogFragment] that gives option to either cancel or exit current profile. */`

Reply:
Done